includes/fields/class-color.php

```diff
@@ -0,0 +1,73 @@
+<?php
+/**
+ * Color field.
+ *
+ * @package HivePress\Fields
+ */
+
+namespace HivePress\Fields;
+
+use HivePress\Helpers as hp;
+
+// Exit if accessed directly.
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Color.
+ */
+class Color extends Text {
+	/**
+	 * Class initializer.
+	 *
+	 * @param array $meta Class meta values.
+	 */
+	public static function init( $meta = [] ) {
+		$meta = hp\merge_arrays(
+			[
+				'label'    => esc_html__( 'Color', 'hivepress' ),
+				'sortable' => false,
```

Comment:
It shouldn't be filterable/searchable either because HEX codes are pretty unique, it's hard to select the exact one unless it's black/white.

Reply:
Make sure that there are no other field settings or parameters that are not needed for Color